### PR TITLE
Improve format, doc and configuration

### PR DESCRIPTION
**Pull request by pombredanne**
This PR applies various format and configuration improvements.
It also refines the documentation

### CODE REVIEW

**Comment by pombredanne** [comment]
@keshav-space do you mind having a look at this?

**Review by keshav-space — APPROVED** [review]
@pombredanne LGTM! test and docs CI is now fixed. We should remove the SCIO documentation from `introduction.rst`.

File: docs/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+The nexB `container-inspector <https://github.com/aboutcode-org/container-inspector>`_ library
+is also a key component of FederatedCode as this tool is used to analyse Docker
+images, containers, root filesystems, and virtual machine images.
+
+.. note::
+    As a common practice, FederatedCode releases usually follow ScanCode Toolkit releases
+    to ensure the latest improvements of the scanning engines are included in the
+    latest release of FederatedCode.
+
```
[the remark this answers is not in the document]
Comment (pombredanne) [reply docs/introduction.rst:108]: Please do!

**Comment by keshav-space** [comment]
Thanks @pombredanne, merging now.

**Pull request merged**